src/codec/index.ts

```diff
@@ -713,14 +683,28 @@ export const TxTypes = {
   "SetMinGasPriceProposal": "/Switcheo.carbon.fee.SetMinGasPriceProposal",
   "RemoveMsgGasCostProposal": "/Switcheo.carbon.fee.RemoveMsgGasCostProposal",
   "RemoveMinGasPriceProposal": "/Switcheo.carbon.fee.RemoveMinGasPriceProposal",
+  "MsgConvertCoin": "/Switcheo.carbon.erc20.MsgConvertCoin",
+  "MsgConvertCoinResponse": "/Switcheo.carbon.erc20.MsgConvertCoinResponse",
+  "MsgConvertERC20": "/Switcheo.carbon.erc20.MsgConvertERC20",
+  "MsgConvertERC20Response": "/Switcheo.carbon.erc20.MsgConvertERC20Response",
+  "MsgUpdateParams": "/Switcheo.carbon.erc20.MsgUpdateParams",
+  "MsgUpdateParamsResponse": "/Switcheo.carbon.erc20.MsgUpdateParamsResponse",
+  "MsgRegisterToken": "/Switcheo.carbon.erc20.MsgRegisterToken",
+  "MsgRegisterTokenResponse": "/Switcheo.carbon.erc20.MsgRegisterTokenResponse",
+  "MsgRegisterERC20": "/Switcheo.carbon.erc20.MsgRegisterERC20",
+  "MsgRegisterERC20Response": "/Switcheo.carbon.erc20.MsgRegisterERC20Response",
+  "MsgUpdateERC20Enabled": "/Switcheo.carbon.erc20.MsgUpdateERC20Enabled",
+  "MsgUpdateERC20EnabledResponse": "/Switcheo.carbon.erc20.MsgUpdateERC20EnabledResponse",
+  "MsgUpdateEVMHookEnabled": "/Switcheo.carbon.erc20.MsgUpdateEVMHookEnabled",
+  "MsgUpdateEVMHookEnabledResponse": "/Switcheo.carbon.erc20.MsgUpdateEVMHookEnabledResponse",
   "MsgMergeAccount": "/Switcheo.carbon.evmmerge.MsgMergeAccount",
   "MsgMergeAccountResponse": "/Switcheo.carbon.evmmerge.MsgMergeAccountResponse",
-  "MsgCreate": "/Switcheo.carbon.btcx.MsgCreate",
-  "MsgCreateResponse": "/Switcheo.carbon.btcx.MsgCreateResponse",
-  "MsgBind": "/Switcheo.carbon.btcx.MsgBind",
-  "MsgBindResponse": "/Switcheo.carbon.btcx.MsgBindResponse",
-  "MsgLock": "/Switcheo.carbon.btcx.MsgLock",
-  "MsgLockResponse": "/Switcheo.carbon.btcx.MsgLockResponse",
+  "MsgCreate": "/Switcheo.carbon.lockproxy.MsgCreate",
+  "MsgCreateResponse": "/Switcheo.carbon.lockproxy.MsgCreateResponse",
+  "MsgBind": "/Switcheo.carbon.lockproxy.MsgBind",
+  "MsgBindResponse": "/Switcheo.carbon.lockproxy.MsgBindResponse",
+  "MsgLock": "/Switcheo.carbon.lockproxy.MsgLock",
+  "MsgLockResponse": "/Switcheo.carbon.lockproxy.MsgLockResponse",
```

Comment:
Have you checked if this is the more used one?

Reply:
this is the correct one, previously got issue